antarest/study/business/utils.py

```diff
@@ -19,6 +20,12 @@
 GENERAL_DATA_PATH = "settings/generaldata"
 
 
+class ClusterType(EnumIgnoreCase):
```

Comment:
@laurent-laporte-pro To be consistent with endpoints, we should use plural

Reply:
We currently have the following endpoints (deprecated endpoints are not listed):

- "/studies/{uuid}/areas/{area_id}/clusters/renewable"
- "/studies/{uuid}/areas/{area_id}/clusters/renewable/{cluster_id}"
- "/studies/{uuid}/areas/{area_id}/clusters/renewable"
- "/studies/{uuid}/areas/{area_id}/clusters/renewable/{cluster_id}"
- "/studies/{uuid}/areas/{area_id}/clusters/renewable"
- "/studies/{uuid}/areas/{area_id}/clusters/thermal"
- "/studies/{uuid}/areas/{area_id}/clusters/thermal/{cluster_id}"
- "/studies/{uuid}/areas/{area_id}/clusters/thermal"
- "/studies/{uuid}/areas/{area_id}/clusters/thermal/{cluster_id}"
- "/studies/{uuid}/areas/{area_id}/clusters/thermal"
- "/studies/{uuid}/areas/{area_id}/storages/{storage_id}"
- "/studies/{uuid}/areas/{area_id}/storages"
- "/studies/{uuid}/areas/{area_id}/storages/{storage_id}/series/{ts_name}"
- "/studies/{uuid}/areas/{area_id}/storages/{storage_id}/series/{ts_name}"
- "/studies/{uuid}/areas/{area_id}/storages/{storage_id}/validate"
- "/studies/{uuid}/areas/{area_id}/storages"
- "/studies/{uuid}/areas/{area_id}/storages/{storage_id}"
- "/studies/{uuid}/areas/{area_id}/storages"
- "/studies/{uuid}/areas/{area_id}/{cluster_type}/{source_cluster_id}"

So, we can consider using the plural form for the `cluster_type`, to be consistent with the other endpoints.